src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy

```diff
@@ -33,6 +37,62 @@ class RoundTripIslSpec extends HealthCheckSpecification {
     via the 'knockoutSwitch' method on the stage env*/
     Integer customWaitOffset = WAIT_OFFSET * 4
 
+    @Unroll
+    @Tidy
+    def "Isl with round-trip properly changes status after port events(#descr)"() {
+        given: "Round-trip ISL with a-switch"
+        def cleanupActions = []
+        def isl = topology.islsForActiveSwitches.find { it.aswitch?.inPort && it.aswitch?.outPort &&
+                [it.srcSwitch, it.dstSwitch].every { it.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD) }
+        }
+        assumeTrue("Wasn't able to find round-trip ISL with a-switch", isl != null)
+        bfd && northboundV2.setLinkBfd(isl)
+
+        when: "Port down event happens"
+        antiflap.portDown(isl.srcSwitch.dpId, isl.srcPort)
+        cleanupActions << { antiflap.portUp(isl.srcSwitch.dpId, isl.srcPort) }
+
+        then: "ISL changed status to FAILED"
+        Wrappers.wait(WAIT_OFFSET * 2) {
```

Comment:
Do we really need to wait `WAIT_OFFSET * 2`?

Reply:
fixed